src/docs/arch-index.dox

```diff
@@ -116,27 +117,25 @@ wt_file -[hidden]right-> log_file
 
 @enduml
 @plantuml_end
+</div>
 
 We go into some detail for some of the internal components.
 
-@subpage arch-glossary
+@subpage arch-dhandle
 
-WiredTiger assigns specific meanings to certain words. Here we decode them.
+An internal structure called Data Handle (dhandle) is used to represent and
```

Comment:
"called a Data Handle". Since this is the same text as before is there anyway to have it in only one place?